go/rtl/auth.go

```diff
@@ -137,11 +137,27 @@ func (s *AuthSession) Do(result interface{}, method, ver, path string, reqPars m
 	defer res.Body.Close()
 
 	if res.StatusCode < 200 || res.StatusCode > 226 {
-		return fmt.Errorf("response error: %s", res.Status)
+		b, err := ioutil.ReadAll(res.Body)
+		if err != nil {
+				return fmt.Errorf("response error. status=%s. error parsing error body", res.Status)
+		}
+
+		return fmt.Errorf("response error. status=%s. error=%s", res.Status, string(b))
```

Comment:
I am wondering if this could break any code that relies on the string.

Reply:
Yeah this could, though consumers _reallllllly_ shouldn't be matching on strings. I'll be implementing better error context in the future so consumers can match on the status code and/or specific errors. https://github.com/looker-open-source/sdk-codegen/issues/795#issuecomment-1069742721

The possibility of breakage is not great, but while we try to fully support the Go SDK, consider it in beta and we will provide best effort to not break downstream dependencies.